skopt/utils.py

```diff
@@ -388,3 +387,61 @@ def point_aslist(search_space, point_as_dict):
     ]
     return point_as_list
 
+def normalize_dimensions(dimensions, estimator="GP"):
```

Comment:
If we go ahead with my idea from https://github.com/scikit-optimize/scikit-optimize/pull/420#issuecomment-313610646 we should remove the `estimator` argument.